sklearn/tests/test_base.py

```diff
@@ -76,8 +77,8 @@ class ModifyInitParams(BaseEstimator):
     Doesn't fulfill a is a
     """
 
-    def __init__(self, a=np.array([0])):
-        self.a = a.copy()
```

Comment:
Looking at the git history: https://github.com/scikit-learn/scikit-learn/commit/680ab517dfec4c60857f503f50b5faf512d37c56 and the original issue: https://github.com/scikit-learn/scikit-learn/pull/5540, the proposed feature in this PR was deprecated and removed in a older version of scikit-learn.